doc/developers/advanced_installation.rst

```diff
@@ -148,6 +148,24 @@ Then you need to set the following environment variables::
 
 Finally you can build the package using the standard command.
 
+FreeBSD
+-------
+
+You first need to have OpenMP library installed::
+
+    sudo pkg install openmp
+    
+This will install headers files in ``/usr/local/include`` and libs in ``/usr/local/lib``.
+Then set the environment variables to these locations::
+
+    export CFLAGS="$CFLAGS -I/usr/local/include"
+    export CXXFLAGS="$CXXFLAGS -I/usr/local/include"
+    export LDFLAGS="$LDFLAGS -L/usr/local/lib -lomp"
+    export DYLD_LIBRARY_PATH=/usr/local/lib
```

Comment:
This feels somewhat hacky cf https://github.com/scikit-learn/scikit-learn/issues/13950#issuecomment-496027032 are those paths not used by default by clang on FreeBSD then? It can be checked with the [following command](https://stackoverflow.com/a/11946295/1791279).

At least, it could be worth mentioning that this is a FreeBSD bug.

Reply:
As far as I understand, clang is currently provided in FreeBSD base without openmp support as of version 12.0. It will be on base when version 12.1 is released in November/2019.

The `openmp` package can be installed, from packages or ports, which installs everything under `/usr/local`, while the default include directories are:

```
/usr/include
/usr/lib/clang/6.0.1/include
 ```

Same with libs.

When FreeBSD 12.1 is released all this becomes obsolete. The package `openmp` has the following annotation:

```
deprecated     : Part of base system since FreeBSD 11.3/12.1
expiration_date: 2019-12-31
```

Reply:
Thanks for these clarifications,

> As far as I understand, clang is currently provided in FreeBSD base without openmp support as of version 12.0. It will be on base when version 12.1 is released in November/2019.

Could you please add some of this context in this PR?

Reply:
Done.